js/app.js

```diff
@@ -11,17 +13,21 @@ if (dogname === 'yes') {
 } else if (dogname === 'no') {
   //console.log('Correct!')
   alert('Correct!');
+  score++;
 }
 //console.log('That is not an answer')
 else {
   alert('That is not an answer!');
 }
-
+}
+dogFunction();
```

Comment:
Try to put all your function calls in one spot at the end, this makes them easier to find to change the order of, and to actually see what your document is doing.